s3file/static/s3file/js/s3file.js

```diff
@@ -48,17 +84,19 @@
     const url = fileInput.getAttribute('data-url')
     const promises = Array.from(fileInput.files).map((file) => {
       const s3Form = new window.FormData()
+      var currFilePart = 0;
       Array.from(fileInput.attributes).forEach(attr => {
         let name = attr.name
         if (name.startsWith('data-fields')) {
           name = name.replace('data-fields-', '')
           s3Form.append(name, attr.value)
         }
       })
+      currFilePart += 1;
       s3Form.append('success_action_status', '201')
       s3Form.append('Content-Type', file.type)
       s3Form.append('file', file)
-      return request('POST', url, s3Form)
+      return request('POST', url, s3Form, currFilePart)
```

Comment:
`currFilePart` will always be `1` since you are setting it to 0 inside the loop.

Reply:
fixed